### PR TITLE
refactor (local helm chart)

### PR DESCRIPTION
**Pull request by ChandanSagar**
- Updated lables as per recent release
- Updated image tags
- Updated release-management.md

Signed-off-by: ChandanSagar <chandan.pradhan@mayadata.io>

### CODE REVIEW

**Comment by akhilerm** [comment]
@ChandanSagar . The seachest probe will be disabled in this release of NDM. Can you change the [ConfigMap](https://github.com/openebs/openebs/blob/dba2d74afc2197db38e3fb81961f1c5553946529/k8s/charts/openebs/templates/cm-node-disk-manager.yaml#L25) to set `seachest-probe` to `false`

File: k8s/charts/openebs/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
-version: 1.0.0.1
+version: 1.1.0.1
```
Comment (kmova) [inline k8s/charts/openebs/Chart.yaml]: Can you re-sync with with master branch. It should be 1.1.0.

File: k8s/charts/openebs/values.yaml [hunk 2]
```diff
@@ -32,10 +32,11 @@ apiserver:
   healthCheck:
     initialDelaySeconds: 30
     periodSeconds: 60
-
+  defaultStorageConfig:
+    enabeled: "false"
```
Comment (kmova) [inline k8s/charts/openebs/values.yaml]: spell check.

Also, can you move this `defaultStorageConfig` section as its own components.

Comment (kmova) [reply k8s/charts/openebs/values.yaml]: Related to this change, you need to make the change in the maya-apiserver deployment. Also need to add this flag into the README.

Comment (prateekpandey14) [reply k8s/charts/openebs/values.yaml]: Also add `OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG` env in Maya-apiserver deployment yaml file and set the value as `false` via this tag.
```
defaultConfigs:
    StorageConfig : 
         enabled: "false"
```

Comment (prateekpandey14) [reply k8s/charts/openebs/values.yaml]: @kmova is `defaultStorageConfig` is true by default ?

File: k8s/charts/openebs/templates/deployment-maya-apiserver.yaml [hunk 3]
```diff
@@ -52,6 +52,8 @@ spec:
           value: "{{ .Values.apiserver.sparse.enabled }}"
         - name: OPENEBS_IO_CSTOR_POOL_SPARSE_DIR
           value: "{{ .Values.ndm.sparse.path }}"
+        - name: OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG
+          value: "{{ .Values.defaultConfigs.storageConfig.enabled }}"
```
Comment (kmova) [inline k8s/charts/openebs/templates/deployment-maya-apiserver.yaml]: Call this `defaultStorageConfig`. This should get included in the values.yaml. There should also be an entry in this yaml to add `OPENEBS_IO_CSTOR_TARGET_DIR ` ENV pointing to `.Values.ndm.sparse.path`

`defaultStorageConfig.enabled` should be documented in readme.

Comment (kmova) [reply k8s/charts/openebs/templates/deployment-maya-apiserver.yaml]: Make sure the ENV variable are available in the operator.yaml also: 
- OPENEBS_IO_CSTOR_POOL_SPARSE_DIR
- OPENEBS_IO_CSTOR_TARGET_DIR
- OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG

Comment (ChandanSagar) [reply k8s/charts/openebs/templates/deployment-maya-apiserver.yaml]: Added the ENV variables. PTAL

File: k8s/charts/openebs/templates/deployment-maya-apiserver.yaml [hunk 4]
```diff
@@ -52,6 +52,10 @@ spec:
           value: "{{ .Values.apiserver.sparse.enabled }}"
         - name: OPENEBS_IO_CSTOR_POOL_SPARSE_DIR
           value: "{{ .Values.ndm.sparse.path }}"
+        - name: OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG
+          value: "{{ .Values.defaultStorageConfig.StorageConfig.enabled }}"
```
Comment (prateekpandey14) [inline k8s/charts/openebs/templates/deployment-maya-apiserver.yaml]: change it to `"{{ .Values.defaultStorageConfig.enabled }}" `

File: k8s/charts/openebs/values.yaml [hunk 5]
```diff
@@ -33,9 +33,13 @@ apiserver:
     initialDelaySeconds: 30
     periodSeconds: 60
 
+defaultStorageConfig:
+  StorageConfig:
+    enabled: "false"
+
```
Comment (prateekpandey14) [inline k8s/charts/openebs/values.yaml]:
```yaml
defaultStorageConfig:
    enabled: "false"
```

File: k8s/charts/openebs/README.md [hunk 6]
```diff
@@ -102,22 +102,23 @@ The following table lists the configurable parameters of the OpenEBS chart and t
...
-| `cstor.volumeMgmt.imageTag`             | Image Tag for cStor Volume Management         | `1.0.0`                                   |
+| `cstor.volumeMgmt.imageTag`             | Image Tag for cStor Volume Management         | `1.1.0`                                   |
 | `policies.monitoring.image`             | Image for Prometheus Exporter                 | `quay.io/openebs/m-exporter`              |
-| `policies.monitoring.imageTag`          | Image Tag for Prometheus Exporter             | `1.0.0`                                   |
+| `policies.monitoring.imageTag`          | Image Tag for Prometheus Exporter             | `1.1.0`                                   |
 | `analytics.enabled`                     | Enable sending stats to Google Analytics      | `true`                                    |
 | `analytics.pingInterval`                | Duration(hours) between sending ping stat     | `24h`                                     |
+| `defaultStorageConfig.enabled`          | Enable default storage class installation     | `false`                                   |
```
Comment (kmova) [inline k8s/charts/openebs/README.md]: Default is `true`

File: k8s/charts/openebs/values.yaml [hunk 7]
```diff
@@ -33,9 +33,12 @@ apiserver:
     initialDelaySeconds: 30
     periodSeconds: 60
 
+defaultStorageConfig:
+  enabled: "false"
```
Comment (kmova) [inline k8s/charts/openebs/values.yaml]: default is `true`

**Pull request merged**